test/integration/api/submissions.js

```diff
@@ -485,6 +485,50 @@ describe('api: /submission', () => {
                 .set('If-None-Match', '"25bdb03b7942881c279788575997efba"')
                 .expect(304)))))));
 
+    it('should successfully save additionally POSTed attachment binary data with s3 enabled', testService((service, { Blobs }) => {
```

Comment:
I remembered some other integration tests:
- downloading form attachments rather than submission attachments (technically tested in e2e test)
- downloading the xlsx form if it was provided (and converted to xml behind the scenes)
- checking what happens if you update or delete a form or submission attachment (old blob could be in S3, new blob could be in DB)

Reply:
Added to TODO.md :+1: